cli_tools/google-osconfig-agent/config/config.go

```diff
@@ -93,13 +94,15 @@ type attributesJSON struct {
 	OSInventoryEnabled string `json:"os-inventory-enabled"`
 	OSPatchEnabled     string `json:"os-patch-enabled"`
 	OSPackageEnabled   string `json:"os-package-enabled"`
+	DebugEnabled       string `json:"debug-enabled"`
```

Comment:
This is a shared namespace, use something like 'os-config-debug'

Reply:
changed... made it consistent throughout with other osconfig attributes.